quarkus/src/main/java/com/example/javaopencommerce/catalog/ItemController.java

```diff
@@ -3,28 +3,39 @@
 import com.example.javaopencommerce.PageDto;
 import com.example.javaopencommerce.catalog.dtos.FullItemDto;
 import com.example.javaopencommerce.catalog.dtos.ItemDto;
+import com.example.javaopencommerce.catalog.dtos.ItemFamilyMemberDto;
 
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
```

Comment:
The second one is not required, as it's covered by first import. 
I'm not for wildcard exports... but let's do either: wildcard it or improt explicit

Reply:
Wont agree here, both are needed, wildcard for one package does not mean that it covers all the sub-packages.